src/shared/components/custom-toast/CustomToast.tsx

```
@@ -0,0 +1,18 @@
+import { type ToastContentProps } from "react-toastify"
+
+
+interface CustomToastProps extends ToastContentProps {
+    text?: string,
+    icon?: React.ReactNode,
```

ESLint

[error] 'React' is not defined.

(no-undef)